.circleci/env-3.6-mpich.yml

```diff
@@ -4,8 +4,8 @@ channels:
   - defaults
 dependencies:
   - python=3.6
-  - dask
-  - distributed
+  - dask>=2.0
+  - distributed>=2.0
```

Comment:
I would expect both `distributed` and `click` to come by default in the `dask` metapackage.